src/buttons/CopyToClipboard/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'preact/hooks'
+import PropTypes from 'prop-types'
+import CSS from './styles.css'
+
+import copyToClipboard from './../../utils/copy-to-clipboard'
+
+const CopyToClipboard = ({
+	dataTestId,
+	valueToCopy,
+	labelToCopy,
+	labelCopying,
+	labelCopied,
+	classToCopy,
+	classCopying,
+	classCopied,
+	copyingFeedbackTime,
+	copiedFeedbackTime,
+}) => {
+	const TO_COPY = { class: classToCopy, label: labelToCopy }
+	const COPYING = { class: classCopying, label: labelCopying }
+	const COPIED = { class: classCopied, label: labelCopied }
+
+	const COPY_TIMEOUT = copyingFeedbackTime
+	const COPIED_TIMEOUT = copiedFeedbackTime
+
+	const [copyStatus, setCopyStatus] = useState(TO_COPY)
+
+	const afterCopy = () => {
+		setCopyStatus(COPIED)
+		setTimeout(() => {
```

Comment:
(minor) this would be a nice place to use `async/await` along with a new helper function for awaiting any delay you give it. This would make for much cleaner/easier-to-read code. Something like: 

```js
const afterCopy = async () => {
  setCopyStatus(copiedClassAndLabel);
  await timeoutDelay(copiedTimeoutInMilliseconds);
  setCopyStatus(toCopyClassAndLabel);
};
```

Let me know if you'd like any help with creating the async `timeoutDelay` helper function that I'm using in the example above.

Reply:
Similar comment below for the `handleOnClick` function.